### PR TITLE
Fixed one issue

### PR DESCRIPTION
**Pull request by MariaMaximova**
For fixing issue #12 and issue #13

### CODE REVIEW

File: converter/converter.py [hunk 1]
```diff
@@ -15,8 +16,13 @@ def celsius_to_fahrenheit(temperature_in_celsius: float) -> float:
     Temp: 20 C
     20 * (9/5) + 32 = 68 F
     """
-    fahrenheit = round(((temperature_in_celsius * (9 / 5)) + 32), 1)
-    return fahrenheit
+    if temperature_in_celsius < ABSOLUTE_ZERO_CEL:
+        quit(f"Error temperature couldn't be lower than {ABSOLUTE_ZERO_CEL}")
```
Comment (OleksiiBulba) [inline converter/converter.py:20]: @MariaMaximova, please use exceptions and add unit tests for raising exception when passing temperature lower than absolute minimum

File: converter/converter.py [hunk 2]
```diff
@@ -25,5 +31,8 @@ def fahrenheit_to_celsius(temperature_in_fahrenheit: float) -> float:
     Temp: 68 F
     (5/9) * (68 - 32) = 20 C
     """
-    celsius = round((5 / 9) * (temperature_in_fahrenheit - 32), 1)
-    return celsius
+    if temperature_in_fahrenheit < ABSOLUTE_ZERO_FAR:
+        quit(f"Error temperature couldn't be lower than {ABSOLUTE_ZERO_FAR}")
```
Comment (OleksiiBulba) [inline converter/converter.py:35]: @MariaMaximova, please use exceptions and add unit tests for raising exception when passing temperature lower than absolute minimum

File: converter/converter.py [hunk 3]
```diff
@@ -25,5 +31,8 @@ def fahrenheit_to_celsius(temperature_in_fahrenheit: float) -> float:
...
+    else:
```
Comment (OleksiiBulba) [inline converter/converter.py:36]: @MariaMaximova, if you use `if` statement that checks for exceptional cases and don't continue the method/function execution you can remove `else` part as it will not be executed anyway if exceptional case occurrs.

File: converter/converter.py [hunk 4]
```diff
@@ -15,8 +16,13 @@ def celsius_to_fahrenheit(temperature_in_celsius: float) -> float:
     Temp: 20 C
     20 * (9/5) + 32 = 68 F
     """
-    fahrenheit = round(((temperature_in_celsius * (9 / 5)) + 32), 1)
-    return fahrenheit
+    if temperature_in_celsius < ABSOLUTE_ZERO_CEL:
+        quit(f"Error temperature couldn't be lower than {ABSOLUTE_ZERO_CEL}")
+#  raise Exception(f"Error values is lower than {ABSOLUTE_ZERO_CEL}")
+
+    else:
```
Comment (OleksiiBulba) [inline converter/converter.py:23]: @MariaMaximova, if you use `if` statement that checks for exceptional cases and don't continue the method/function execution you can remove `else` part as it will not be executed anyway if exceptional case occurrs.

File: main.py [hunk 5]
```diff
@@ -18,23 +27,51 @@
     help="Take temperature in Celsius and convert to Fahrenheit"
 )
 
+# parser.add_argument(
```
Comment (OleksiiBulba) [inline main.py:30]: @MariaMaximova, it is bad practice to leave commented code.
Please uncomment it and use or remove it.
I suggest uncommenting and trying to get it work as intended to fix the issue with both flags (-c and -f)

File: main.py [hunk 6]
```diff
@@ -18,23 +27,51 @@
...
+#     help="use flag -f(--fahrenheit) and provide one value for converting"
+# )
+#
+# scale_group.add_argument(
+#     "-c", "--celsius", action="store_true",
+#     help="use flag -c(--celsius) and provide one value for converting"
+# )
+
 args = parser.parse_args()
+# print(sys.argv)
```
Comment (OleksiiBulba) [inline main.py:48]: @MariaMaximova, I suggest using `--verbose` argument for print it: if the argument is present, then you can print this debug info.

File: main.py [hunk 7]
```diff
@@ -18,23 +27,51 @@
...
+    while answer not in (CELSIUM_INPUT + FAH_INPUT + QUIT_INPUT):
         answer = str(input("What temperature you want convert?: ")).lower()
-        if answer in QUIT_INPUT:
-            quit()
+
+    if answer in QUIT_INPUT:
+        quit()
 
     if answer in CELSIUM_INPUT:  # Convert from Celsius to Fahrenheit
         celsius = input("Provide temperature in Celsius: ")
```
Comment (OleksiiBulba) [inline main.py:77]: @MariaMaximova, it is recommended to add 
```
if __name__ == '__main__':
```
to your code so if I import your main file it will not be excecuted and will not yield any strange errors.

File: main.py (hunk 6 above)
Comment (OleksiiBulba) [reply main.py:48]: @MariaMaximova, you can use logging package for logging (do not forget to setup logging level)
